data/templates/default.t2t

```diff
@@ -0,0 +1,19 @@
+$pagetitle$
+$author-meta$
+$date-meta$
+
+
+
```

Comment:
This is why you're getting all that blank space at the beginning.  Do you want the 3 blank lines if there's a title?  If there's no title block, would it be better to omit them?  You can use conditionals like
```
$if(pagetitle)$
$pagetitle$
$author-meta$
$date-meta$


$endif$
```